src/messaging/ExchangeContext.h

```diff
@@ -214,6 +214,47 @@ class DLL_EXPORT ExchangeContext : public ReliableMessageContext,
      */
     bool IsSendExpected() const { return mFlags.Has(Flags::kFlagWillSendMessage); }
 
+    /**
+     * Notifies the exchange that our consummer is going to send a message where it knows the peer (receiver) should be active
+     *
+     * This API is used to set the ShouldPeerBeActive flag.
+     * The ShouldPeerBeActive flag allows the consummer of the exchange to notify the exchange that the peer (receiver)
+     * will be active for the next message the consummer will send.
+     * This allows the consummer to notify the exchange on the state of the peer with the context of the messages being sent.
+     *
+     * The ShouldPeerBeActive flag is only used when calculating the next retransmission time in the ReliableMessageMgr.
+     * If the ShouldPeerBeActive flag is true, we will used the Active Retransmission timeout when calculating the next
+     * retransmission time. If the ShouldPeerBeActive flag is false, we will rely on other information to determine which
+     * retransmission timeout should be used. See the ShouldPeerBeActive API for the alternative means to determine if the Peer
+     * (receiver) is active.
+     *
+     * The first message of the exchange set as the initiator should never have the ShouldPeerBeActive flag to true.
+     * The API should only called after receiving a message from the peer and after calling the WillSendMessage API.
```

Comment:
"should only called" --> "should only be called"

Reply:
done